dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -1162,3 +1190,57 @@ void CommandBarFlyoutCommandBar::ClearShadow()
         }
     }
 }
+
+bool CommandBarFlyoutCommandBar::IsExpandedUp()
+{
+    return m_isExpandedUp;
+}
+
+bool CommandBarFlyoutCommandBar::HasSecondaryOpenCloseAnimations()
+{
+    return SharedHelpers::IsAnimationsEnabled() && static_cast<bool>(m_expandedDownToCollapsedStoryboard || m_expandedUpToCollapsedStoryboard || m_collapsedToExpandedUpStoryboard || m_collapsedToExpandedDownStoryboard);
+}
+
+void CommandBarFlyoutCommandBar::AttachEventsToSecondaryStoryboards()
+{
+    winrt::IControlProtected thisAsControlProtected = *this;
+
+    const auto addDropShadowFunc = [this](auto const&, auto const&)
+    {
+        if (SharedHelpers::IsAnimationsEnabled())
+        {
+            if (auto owningFlyout = m_owningFlyout.get())
```

Comment:
>auto [](start = 16, length = 4)

nit: when getting values from trackerrefs using auto&& is slightly more efficient. (Using this assumes that the value in the tracker ref wont change while you have the local)